### PR TITLE
This is the right way to organized the project

### PR DESCRIPTION
**Pull request by Wabri**
This is what I've done in my fork:
- I took the liberty to change the place of the maven project (it is important to divide the project from the repository main directory)
- Update pom
- Edited the coordinates of maven project
- Deleted some useless file

Commit by Commit I explained it more accurately (you can see from the commit view in PR what I actually changed).

Don't merge unless you understand what I've done and feel free to ask (using the pull request review mode to add comments directly where it's the problem).

I will create a branch and work to review the entire code...

### CODE REVIEW

File: Inventory.System.Store/.classpath [hunk 1]
```diff
@@ -12,7 +12,7 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/J2SE-1.5">
```
[the remark this answers is not in the document]
Comment (ajfmo) [reply Inventory.System.Store/.classpath]: Oh, I see, the classpath configuration updates automatically when restructured the project, right?

[the remark this answers is not in the document]
Comment (Wabri) [reply Inventory.System.Store/.classpath]: This is the metadata file of .classpath, classpathentry it's a argument which is passed to compiler of JVM (java virtual machine) and indicate the version of java enviroment.

[the remark this answers is not in the document]
Comment (Wabri) [reply Inventory.System.Store/.classpath]: This change when update the plugin maven-compiler to 1.8

[the remark this answers is not in the document]
Comment (ajfmo) [reply Inventory.System.Store/.classpath]: As you can see my fellow, I'm a total beginner in this business, I really appreciate very much your time! Thanks a lot!

**Pull request merged**